ci/docs-builder.py

```diff
@@ -141,7 +141,7 @@ def main():
 
     # Get the latest api.yaml file to build the rest api docs
     if is_pulp3:
```

Comment:
@bmbouter is this file being used to build pulp3 docs? (if not can we remove it?)